pkg/armhelpers/azureclient.go

```diff
@@ -138,13 +138,8 @@ func NewAzureClientWithClientSecret(env azure.Environment, subscriptionID, clien
 	return getClient(env, subscriptionID, armSpt, adSpt)
 }
 
-// NewAzureClientWithClientCertificate returns an AzureClient via client_id and jwt certificate assertion
-func NewAzureClientWithClientCertificate(env azure.Environment, subscriptionID, clientID, certificatePath, privateKeyPath string) (*AzureClient, error) {
-	oauthConfig, _, err := getOAuthConfig(env, subscriptionID)
-	if err != nil {
-		return nil, err
-	}
-
+// NewAzureClientWithClientCertificateFile returns an AzureClient via client_id and jwt certificate assertion
```

Comment:
Now i cannot use the comment to differentiate the two methods. They are exactly the same.